📄🔧：update README.md (#81)
OpenINF/docker-fisher: README.md

<br />

As pull requests are welcome, if one would like to make significant

> changes, preferably, please do discuss the shift first (intern).

Author: ```suggestion
changes, preferably, please do discuss the shift first (in-tern).
```

> changes, it is preferable to discuss the change first. Likewise, 

Author: ```suggestion
changes, it is preferable to discuss the change first. Likewise.
```

> If something needs fixing but you need help to do it yourself,

Author: ```suggestion
if something needs fixing, but you need help to do it yourself,
```
